### DISCUSSION TITLE
10xGenomics Cell rangerの使用について

### DISCUSSION THREAD

**Discussion by kz-ym** [nig-sc/nigsc_homepage2#311] [Q&A] [answered]
遺伝研スパコンで10xGenomics社のCell rangerをインストールして使うことはできますでしょうか？
もしインストール方などを把握されていたらお教えいただけますと幸いです。
よろしくお願いいたします。

**Comment by sc-fs** [comment, marked as answer]
自身のローカル環境にインストールして利用されているユーザは既にいらしゃいます。
インストール手順は、10xのcellrangerのページを参考にして下さい。
[https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/tutorial_in#download](https://www.google.com/url?q=https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/tutorial_in%23download&source=gmail-imap&ust=1679889598000000&usg=AOvVaw3Jc7oYWU-u6o36qBxofZ6Y)
